src/Components/NavigationCard.css

```diff
@@ -1,5 +1,14 @@
 a.nav-link {
-   padding: .5rem;
+   padding: .4rem;
+   /* font-family: 'Pinyon Script', cursive; */
```

Comment:
If this isn't needed, you might want to take this out. Seems like you're opting for the different font-style below.